index.js

```diff
@@ -487,8 +487,7 @@ class Spinnies {
         currentSpinner.options.status = newStatus;
       }
     });
-    this.checkIfActiveSpinners();
```

Comment:
From testing on a fork of the parent "spinnies" project, this, and the other call to `this.checkIfActiveSpinners` should be left in.